packages/compass-collection/src/modules/stats.ts

```diff
@@ -0,0 +1,126 @@
+import type { AnyAction } from 'redux';
+import numeral from 'numeral';
+
+/**
+ * The prefix.
+ */
+const PREFIX = 'collection';
+
+/**
+ * Collection details updated action name.
+ */
+export const UPDATE_COLLECTION_DETAILS = `${PREFIX}/stats/UPDATE_COLLECTION_DETAILS`;
+
+/**
+ * Collection details reset action name.
+ */
+export const RESET_COLLECTION_DETAILS = `${PREFIX}/stats/RESET_COLLECTION_DETAILS`;
+
+/**
+ * Invalid stats.
+ */
+const INVALID = 'N/A';
+
+export interface StatsObject {
```

Comment:
Small idea or suggestion, since this `StatsObject` looks like it always pertains to the stats of a collection, how do you feel about the name `CollectionStats`?
```suggestion
export interface CollectionStats {
```

Reply:
It will overlap with the component name then, but I will make it `CollectionStatsObject` to be more explicit.